system-addon/content-src/components/Search/_Search.scss

```diff
@@ -122,6 +123,21 @@
       }
     }
 
+    .contentSearchOneOffItem {
+      background-image: none;
```

Comment:
Not sure why images were being used as dividers, that code must be super old. Hacking over it here with plans to fix it in m-c when we update `contentSearchUI` for other coming changes (change the settings button to a gear icon like the awesome bar). It does make the hover background color not fill the space vertically but good enough for now says Amy.